src/frame.cc

```diff
@@ -1066,14 +1066,15 @@ napi_value getFrameData(napi_env env, napi_callback_info info) {
   ref = f->frame->buf[0] ? av_buffer_ref(f->frame->buf[0]) : nullptr;
   size = ref ? ref->size : 0;
   curElem = 0;
+  void* resultData;
   // work through frame bufs checking whether allocation refcounts are shared
   for ( int x = 1 ; x < AV_NUM_DATA_POINTERS ; x++ ) {
     // printf("Buffer %i is %p\n", x, f->frame->data[x]);
     if (f->frame->data[x] == nullptr) continue;
     size_t bufSize = size;
     if (f->frame->buf[x] == nullptr)
       bufSize = f->frame->data[x] - f->frame->data[x-1];
-    status = napi_create_external_buffer(env, bufSize, data, frameBufferFinalizer, ref, &element);
+    status = napi_create_buffer_copy(env, bufSize, data, &resultData, &element);
```

Comment:
I noticed you've removed the finalizers. Is there something we should still do for those? It seems like those are called when the objects are garbage collected. Do we not need to clean up the same way anymore?

Reply:
I guess this is what you mean in the description for 
> I'm not sure about memory leaks.